### PR TITLE
Implement crs and geometrycolumn fallbacks via DataAPI

### PR DESCRIPTION
**Pull request by asinghvi17**
- [x] Needs tests
- [x] needs docs

### CODE REVIEW

File: src/fallbacks.jl [hunk 1]
```diff
@@ -95,8 +95,17 @@ issimple(t::AbstractMultiPointTrait, geom) = allunique((getgeom(t, geom)))
...
+crs(::AbstractTrait, geom) = _get_dataapi_metadata(geom, "GEOINTERFACE:crs", nothing)
+
+function _get_dataapi_metadata(geom::GeomType, key, default) where GeomType
+    if DataAPI.metadatasupport(GeomType).read
+        if key in DataAPI.metadatakeys(geom)
+            return DataAPI.metadata(geom, key; style = false)
+        end
+    end
+    return default
+end
```
Comment (asinghvi17) [inline src/fallbacks.jl:108]: Should I move this to some other location / file?

Comment (evetion) [reply src/fallbacks.jl:108]: Maybe to (a new) metadata.jl? I discussed having a Geometadata.jl package with @rafaqz, but that's probably overkill for now.

**Comment by rafaqz** [comment]
You mean if its a single `String` we should return ` (Symbol(str),)`

**Comment by asinghvi17** [comment]
Yes

**Comment by rafaqz** [comment]
Sounds pragmatic

**Comment by asinghvi17** [comment]
OK, I've added some aftercare functions here.  Do we want anything else?

**Review by rafaqz — APPROVED** [review]
LGTM. Can we merge?

**Pull request merged**